.github/workflows/bin/run-local-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+# shellcheck disable=SC1091
+source "$GITHUB_WORKSPACE"/bin/helpers.sh
+
+chmod +x "$GITHUB_WORKSPACE"/test_proj/bin/*.sh
```

Comment:
Actually, I think this is the result of sourcing this from the repository as a plugin. While _git_ would track the permissions, I don't think _Composer_ makes the files executable, because I ran into the same thing locally.

Reply:
I'm considering adding a note to the README about this behavior or adding something to the composer plugin, but I need to run this on a couple other projects to know for certain if I'm right.